packages/backend/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/app.js",
+  "scripts": {
+    "build": "tsc",
+    "serve": "node .",
+    "start": "npm run build && npm run serve",
```

Comment:
What do you think about using something like ts-node-dev or nodemon to automatically restart app on files changes?